tailwind.config.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
       "base-400": "var(--base-400)",
       "base-500": "var(--base-500)",
     },
```

Comment:
I think you need to add more here, As you can see the footer before and after your PR. I think some colors as grey changes to black as it was not present in config..

Reply:
Don't worry, even after adding colors to theme all others tailwind colors can be used just we have to put them in square brackets.

Reply:
And after using button component in footer it will be okay.

Reply:
> Don't worry, even after adding colors to theme all others tailwind colors can be used just we have to put them in square brackets.

Oh okay! Correct.

Reply:
Yes, and for increased font size, it was important to be changed else the rem units were not working as normally expected in tailwind.
1rem was equal to approx 10 rather than 16 before the change.

Reply:
oh no,  okay got that!